.github/labeler.yml

```
@@ -0,0 +1,38 @@
+# Configuration for GitHub Action `.github/workflows/pull-request-labeler.yml`.
+# See https://github.com/marketplace/actions/labeler for more information.
+
+# Only documentation changes
+docs:
+  - all: [ '**/*.md' ]
+
+# Only test changes
+dev/testing:
+  - all: [ '{spec/**,integration/**}' ]
+
+# Changes to Profiling
+profiling:
+  - any: [ 'lib/datadog/profiling/**' ]
```

Minor: Add the C stuff in as well

```suggestion
profiling:
  - any: [ '{lib/datadog/profiling/**,ext/ddtrace_profiling_loader/**,ext/ddtrace_profiling_native_extension/**}' ]
```